WordPress/src/main/res/layout/login_2fa_screen.xml

```diff
@@ -1,77 +1,28 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
```

Comment:
This `xmlns:app="http://schemas.android.com/apk/res-auto"` is no longer used.  It can be removed.